public/app/plugins/datasource/cloudwatch/components/SQLBuilderEditor/utils.ts

```diff
@@ -146,6 +146,8 @@ export function setSql(query: CloudWatchMetricsQuery, sql: SQLExpression): Cloud
 
 export function setNamespace(query: CloudWatchMetricsQuery, namespace: string | undefined): CloudWatchMetricsQuery {
   const sql = query.sql ?? {};
+  //updating `namespace` props for CloudWatchMetricsQuery
+  query.namespace = namespace ? namespace : '';
```

Comment:
_Note :_ I added namespace here compared to existing code as it seems they could be out of sync if we only changed the namespace from the sql query (i.e. we could have a CloudWatchMetricsQuery with namespace: 'ec2' and sql.select.from.property.name : 'athena').

It was not breaking anything as the SQLBuilderEditor code gets its namespace from the sql query only but it seemed bad practice to leave the old namespace somewhere?

Any thoughts?